task10.py

```diff
@@ -0,0 +1,89 @@
+'''
+Print the following patterns. In the patterns ‘_’ (underscore) denotes the space. **_**
+1.
+    ** **
+    *   *
+        
+    *   *
+    ** **
+2.
+      *  
+     *** 
+    *****
+     *** 
+      *  
+3.
+    *
+    **
+    * *
+    *  *
+    *   *
+    ******
+4.
+    *****
+    *   *
+    *   *
+    *   *
+    *****
+5.
+    1 
+    2 3 
+    4 5 6 
+    7 8 9 10 
+    11 12 13 14 15  
+'''
+#1
+n=int(input("Enetr a number :- "))
+
+#pattren 1 and 2 convert even to odd 
+num=0
+if n%2 ==0:
+    num=+n+1
```

Comment:
Please use correct syntax here. `num=+n+1` is not the correct. However it looks correct, but it makes your code looks complex.